src/Symfony/Component/Uuid/phpunit.xml.dist

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/5.2/phpunit.xsd"
+         backupGlobals="false"
+         colors="true"
+         bootstrap="vendor/autoload.php"
+         failOnRisky="true"
+         failOnWarning="true"
+>
+    <php>
+        <ini name="error_reporting" value="-1" />
+    </php>
+
+    <testsuites>
+        <testsuite name="Symfony UUID Component Test Suite">
+            <directory>./Tests/</directory>
+        </testsuite>
+    </testsuites>
+
+    <filter>
+        <whitelist>
+            <directory>./</directory>
+            <exclude>
+                <directory>./Tests</directory>
+                <directory>./vendor</directory>
```

Comment:
```suggestion
                <directory>./Tests/</directory>
                <directory>./vendor/</directory>
```

Reply:
Why? I took it from string components

Reply:
Yes, but it's inconsistent.

In line 17 we use a trailing slash, here we don't.

Using a trailing slash is not necessary, but to the reader it indicates that we are referencing a directory.